docs/processors/getting-started.mdx

```diff
@@ -1,6 +1,7 @@
 ---
 title: 'Getting Started with Processors'
```

Comment:
It seems that this was written when we didn't yet have configuration files since we just mention how processors are created in the API. I think it would be nice to rename "Adding and configuring a processor" to something like "Creating processors through the API" and add a similar section for the configuration file. 

Sidenote: this can be done in a separate PR, as it doesn't seem in scope of this one, although I won't stop you if you want to squeeze it in.